packages/jest-config/src/normalize.ts

```diff
@@ -489,6 +489,27 @@ export default async function normalize(
   hasDeprecationWarnings: boolean;
   options: AllOptions;
 }> {
+  // Process plugin config hooks before validation
+  if (initialOptions.plugins && initialOptions.plugins.length > 0) {
```

Comment:
Can `initialOptions.plugins` be `null` or `undefined`? If yes, you can use optional chaining syntax `initialOptions.plugins?.length > 0`